GTG/gtk/editor/editor.py

```diff
@@ -505,7 +583,7 @@ def change_status(self, widget):
     def delete_task(self, widget):
         # this triggers the closing of the window in the view manager
         if self.task.is_new():
-            # self.req.delete_task(self.task.get_id())
+            self.req.delete_task(self.task.get_id())
```

Comment:
to be honest? By a mistake :smiley_cat: I just noticed it and can't find a reason for me doing that (or for someone else leaving it commented out!)

Hmm, apparently it was not needed before